### ISSUE TITLE
.DS_STORE

### ISSUE THREAD

**Issue by vsoch** [ENCODE-DCC/pipelines-metadata#1]
Was .DS_STORE intentionally added?

**Comment by anacis** [comment]
No, it was automatically added when I made the folders. Could you explain
to me what it does?

**Comment by vsoch** [comment]
(I believe) that `DS_STORE` is a Mac only little database that is stored in folders, and it's hidden so you only see it show up in repos given that the user did `git add *`. I would remove it from Github (e.g., if I were to clone this to my Ubuntu machine, this would not be needed), and generally it's good practice to not do a `git add *` of any kind, because this is how you accidentally add hidden (or other) files with credentials, etc. I did a little analysis across a bunch of repos from software publications and just counted how often this happens, and unfortunately it's pretty often, see the green bar plot in this writeup --> https://vsoch.github.io/2018/extension-counts/
